Automattic/sensei
# Add tests for email templates setup

Resolves #6509 

## Proposed Changes
* Add tests for classes introduced earlier in https://github.com/Automattic/sensei/pull/6548
* Rename Template_Wizard to Email_Seeder, Email_Data to Email_Seeder_Data.
* Remove some temporary code.

## Testing Instructions

1. Enable the feature flag: `add_filter( 'sensei_feature_flag_email_customization', '__return_true' );`
2. Run `npm run test-php`.
3. Go to Sensei LMS -> Tools and use Re-create Emails.
4. Go to Sensei LMS -> Settings -> Emails.
5. Check you can see emails, try switching tabs (Student's, Teacher's).

## New/Updated Hooks

* `sensei_emails_email_data` -> `sensei_emails_seeder_data` (Renamed).

## Deprecated Code


*

## Pre-Merge Checklist
